components/infobox/commons/infobox_person.lua

```diff
@@ -144,7 +145,13 @@ function Person:createInfobox()
 				}}
 			end}
 		}},
-		Cell{name = 'Alternate IDs', content = {args.ids or args.alternateids}},
+		Cell{name = 'Alternate IDs', content = {
+				table.concat(
+					Array.map(mw.text.split(args.ids or args.alternateids or '', ',', true), mw.text.trim),
```

Comment:
```suggestion
					Array.map(mw.text.split(args.ids or args.alternateids or '', ',', true), function(id) return mw.text.trim(id) end),
```
since `Array.map` gives the index back to the function as 2nd param and `mw.text.trim` supports a 2nd param `charset`.
> If `charset` is supplied, it should be a string as appropriate to go inside brackets in a [Ustring pattern](https://www.mediawiki.org/wiki/Extension:Scribunto/Lua_reference_manual#Ustring_patterns), i.e. the "set" in [set]. The default charset is ASCII whitespace, `"\t\r\n\f "`